packages/frontend/src/components/Header.jsx

```diff
@@ -17,10 +17,10 @@ export default observer(() => {
     const val = BigInt(`0x${searchInput.replace('0x', '')}`)
     if (val > BigInt(2) ** BigInt(160)) {
       // it's an epoch key
-      navigate(`/user/${searchInput})`)
+      navigate(`/epochKey/${searchInput}`)
     } else {
       // otherwise treat it as an attester id
-      navigate(`/attester/${searchInput})`)
+      navigate(`/attester/${searchInput}`)
```

Comment:
@vimwitch 
atm `attesterId` in search bar renders AttesterPage, `epochKey` AND `userId` in search bar render EpochKeyPage. How can we know if searchInput is a `userId`?

Reply:
So we can tell if a search input is an `attesterId` by checking if the numeric value is < 2**160. But `epochKey` and `userId` are roughly the same size. To tell the difference we need to ask the server whether the input is an `epochKey` or `userId` (or neither). The server will check if it has seen the identifier in either a signup event, or an attestation event.